client/pkg/omnictl/root.go

```diff
@@ -28,11 +30,23 @@ var RootCmd = &cobra.Command{
 
 func init() {
 	RootCmd.PersistentFlags().StringVar(&access.CmdFlags.Omniconfig, "omniconfig", "",
-		fmt.Sprintf("The path to the omni configuration file. Defaults to '%s' env variable if set, otherwise the config directory according to the XDG specification.",
+		fmt.Sprintf("The path to the omni configuration file. Defaults to '%s' env variable if set, otherwise '%s'. "+
+			"'%s' is Deprecated and only used as a last resort for reading existing configuration file.",
 			config.OmniConfigEnvVar,
+			filepath.Join("$HOME", constants.TalosDir, config.OmniRelativePath),
+			filepath.Join("$XDG_CONFIG_HOME", config.OmniRelativePath),
 		))
 	RootCmd.PersistentFlags().StringVar(&access.CmdFlags.Context, "context", "",
 		"The context to be used. Defaults to the selected context in the omniconfig file.")
 	RootCmd.PersistentFlags().BoolVar(&access.CmdFlags.InsecureSkipTLSVerify, "insecure-skip-tls-verify", false,
 		"Skip TLS verification for the Omni GRPC and HTTP API endpoints.")
+	RootCmd.PersistentFlags().StringVar(
+		&access.CmdFlags.SideroV1KeysDir,
+		"siderov1-keys-dir",
```

Comment:
I feel like this flag name could be confusing, users don't know much about the siderov1 thing. Maybe do simply "keys-dir" instead?